### PR TITLE
Added new classes, MPTT, extend structure, fixes.

### PR DESCRIPTION
**Pull request by NickiHell**
So work....

### CODE REVIEW

**Comment by NickiHell** [comment]
???Review???

File: diagnosis/models.py [hunk 1]
```diff
@@ -13,4 +17,67 @@ def __str__(self):
...
+
+    def __str__(self):
+        return self.text
+
+    class Meta:
+        verbose_name = 'question'
+        verbose_name_plural = 'questions'
+
+
+class AnswerMap(models.Model):
```
Comment (de10rean) [inline diagnosis/models.py]: In this case foreign keys must have CASCADE ```on_delete``` behavior

```class Meta```?
```__str__```?
```verbose_names```?

File: templates/base.html [hunk 2]
```diff
@@ -20,19 +20,14 @@
...
 
-<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
-        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
-        crossorigin="anonymous"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
-        integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
-        crossorigin="anonymous"></script>
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
 <script src="../static/bootstrap/js/bootstrap.min.js"></script>
```
Comment (de10rean) [inline templates/base.html]: Pls, change to ```static``` templatetag

**Comment by NickiHell** [comment]
Wow, such many mistakes.... @de10rean

File: diagnosis/models.py [hunk 3]
```diff
@@ -13,4 +17,67 @@ def __str__(self):
...
+
+    class MPTTMeta:
+        order_insertion_by = ['name']
+
+
+#DiagnosisSignature model
+class DiagnosisSignature(models.Model):
+    diagnosis = models.ForeignKey(Diagnosis, on_delete='NULL')
+    symptom = models.ForeignKey(Symptom, on_delete='NULL')
+    symptom_type = models.PositiveSmallIntegerField('symptom_type', choices=[])
```
[the remark this answers is not in the document]
Comment (NickiHell) [reply diagnosis/models.py]: Здесь вот нихуя не понимаю, между чем выбираем?

[the remark this answers is not in the document]
Comment (NickiHell) [reply diagnosis/models.py]: `YEAR_IN_SCHOOL_CHOICES = (
    ('FR', 'Freshman'),
    ('SO', 'Sophomore'),
    ('JR', 'Junior'),
    ('SR', 'Senior'),
)`
????

File: templates/base.html (hunk 2 above)
Comment (NickiHell) [reply templates/base.html]: Not yet maybe...

**Pull request merged**